htdocs/core/triggers/interface_20_modWorkflow_WorkflowManager.class.php

```diff
@@ -80,24 +81,22 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 						setEventMessages($langs->trans("OrderExists"), null, 'warnings');
 					}
 					return $ret;
-				} else {
-					include_once DOL_DOCUMENT_ROOT.'/commande/class/commande.class.php';
-					$newobject = new Commande($this->db);
+				}
 
-					$newobject->context['createfrompropal'] = 'createfrompropal';
-					$newobject->context['origin'] = $object->element;
-					$newobject->context['origin_id'] = $object->id;
+				include_once DOL_DOCUMENT_ROOT.'/commande/class/commande.class.php';
+				$newobject = new Commande($this->db);
 
-					$ret = $newobject->createFromProposal($object, $user);
-					if ($ret < 0) {
-						$this->error = $newobject->error;
-						$this->errors[] = $newobject->error;
-					}
+				$newobject->context['createfrompropal'] = 'createfrompropal';
+				$newobject->context['origin'] = $object->element;
+				$newobject->context['origin_id'] = $object->id;
 
-					$object->clearObjectLinkedCache();
+				$ret = $newobject->createFromProposal($object, $user);
+				if ($ret < 0)
```

Comment:
Try to always use { on condition, even if one line

Reply:
Ok no problem.